tensorflow/python/keras/layers/pooling.py

```diff
@@ -480,7 +480,7 @@ class AveragePooling2D(Pooling2D):
     - If `data_format='channels_first'`:
       4D tensor with shape `(batch_size, channels, pooled_rows, pooled_cols)`.
   
-  >>>Average_pooling_layer = tf.keras.layers.AveragePooling2D(pool_size = (2,2),strides = (1,1),padding = "same")
+  >>>average_pooling_layer = tf.keras.layers.AveragePooling2D(pool_size = (2,2),strides = (1,1),padding = "same")
```

Comment:
Thanks @generationXcode for fixing some of style issues. Not sure if this is necessary, but I think there may need to be a space after >>> . All of @WilliamHYZhang 's requested changes also lgtm. Cheers and keep it up!

Reply:
sure

Reply:
Dit what you said😊

Reply:
Nice job 😃